### PR TITLE
Race Trait Adaption

### PR DESCRIPTION
**Pull request by zumbak04**
## Changelog:
- Added all the race traits back (so far no effects, just placeholders).
- Every race trait has localisation and icon.

## How to test:
Enter the game, open the Character Finder, check character traits. Tell your opinion on icons and texts.

If you're a @Warcraft-GoA-Development-Team/english-watchers, please proofread the `.yml` files.

### CODE REVIEW

File: localization/english/wc_traits_l_english.yml [hunk 1]
```diff
@@ -0,0 +1,1027 @@
...
+ trait_creature_gnoll_character_desc:0 "$trait_creature_gnoll_desc$"
+ trait_creature_murloc:0 "Murloc"
+ trait_creature_murloc_desc:0 "Murlocs are a strange combination of humanoid and fish, an ancient two-legged race of amphibians. They live along the coasts and within the depts of oceans, the coasts of lakes and rivers, as well as in underwater ruins. Murlocs are capable of breathing on land, although they rarely risk moving too far from water. Their hunched appearance is rather awkward, but don't let this mislead you - some murlocs travel overland with great speed and agility. Murlocs do not care about individuality or creativity, as this does not help the survival of the tribe or most of their race. They value obedience above all else. Agility and cruelty are also important to them, reflected in their hunting skills and strength."
+ trait_creature_murloc_character_desc:0 "$trait_creature_murloc_desc$"
```
Comment (arithon) [inline localization/english/wc_traits_l_english.yml:10]: Why do we have "This character is a human", but not in the other trait descriptions?

File: localization/english/wc_traits_l_english.yml [hunk 2]
```diff
@@ -0,0 +1,1027 @@
...
+ trait_being_undead:0 "Undead"
+ trait_being_undead_desc:0 "The literal meaning of undead is simple - a creature or person risen from death. The undoing of death, hence undead. While they vary in size, shape, and form, the most common undead are those who have risen from their own (or other's) accord, who maintain their sanity and memories from life. This character in question is a textbook undead, and may act differently from every other undead creature due to their own autonomy."
+ trait_being_undead_character_desc:0 "$trait_being_undead_desc$"
+ trait_creature_ghoul:0 "Ghoul"
+ trait_creature_ghoul_desc:0 "Men who have finally been given the grace of undeath are known as ghouls. Varying in size, shape, and form, ghouls are the shocktroopers of every undead mass. Using speed as their advantage in battle, these charging hulks of fallen warriors are every shieldbearer's worst nightmare."
+ trait_creature_ghoul_character_desc:0 "$trait_creature_ghoul_desc$"
+ trait_creature_wraith:0 "Spirit"
+ trait_creature_wraith_desc:0 "This character is an incorporeal spirt."
+ trait_creature_wraith_character_desc:0 "$trait_creature_wraith_desc$"
+ trait_lich_king:0 "Lich [Root.GetQueenKing]"
```
Comment (arithon) [inline localization/english/wc_traits_l_english.yml:434]: Does the scope still work?

**Review by Nelpok — COMMENTED** [review]
Seems all ancients are lacking on their traits

<details>

<summary>Click to expand</summary>

image
image

</details>

Another thing that i noticed is the ammount of text there is on some traits for example humans have a massive wall of text of information, while others are too short, like night elves, or are directly laking information about them, like pandaren. I find this could be problematic because you also have to add the race modifiers (Opinion, Stat changes, etc) and it could be left...to big or just lacking in general

<details>

<summary>Click to expand</summary>

image
image
image

</details>

I also found those under creature that lack an image 

<details>

<summary>Click to expand</summary>

image

</details>

All images for their races are GREAT

File: localization/english/wc_traits_l_english.yml (hunk 1 above)
Comment (zumbak04) [reply localization/english/wc_traits_l_english.yml]: > Why do we have "This character is a human", but not in the other trait descriptions?

`_character_desc` shows when the trait is on character. `_desc` shows when the trait is in an event option for example.

File: localization/english/wc_traits_l_english.yml (hunk 2 above)
Comment (zumbak04) [reply localization/english/wc_traits_l_english.yml:434]: > Does the scope still work?

Definitely not, I'll adjust it once the trait is here. So far, only races.

**Comment by zumbak04** [comment]
> Seems all ancients are lacking on their traits

Fixed in ff56bcb914c543e0632c24b5629cf2f321652c00.

> Another thing that i noticed is the ammount of text there is on some traits for example humans have a massive wall of text of information, while others are too short, like night elves, or are directly laking information about them, like pandaren. I find this could be problematic because you also have to add the race modifiers (Opinion, Stat changes, etc) and it could be left...to big or just lacking in general

Changed in 6603e2009e6e3228420d6ca84484b4d0550f5341, changed in e3e81ec7a074eac86bfc2258c592b144d3bd10af.

> I also found those under creature that lack an image

Fixed in e0acdc45feaafdc31e71f8d90c520e3e6d5458e1.

File: localization/english/wc_traits_l_english.yml (hunk 1 above)
Comment (arithon) [reply localization/english/wc_traits_l_english.yml]: Wouldn't it make more sense to put "This character is a ..." in all "trait_creature_..._character_desc" lines?

Comment (arithon) [reply localization/english/wc_traits_l_english.yml]: Nvm, forget my previous question, I got it the wrong way.

**Review by arithon — APPROVED** [review]
racial traits lgtm

File: common/traits/wc_race_traits.txt [hunk 3]
```diff
@@ -0,0 +1,1488 @@
...
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_magnataur_desc
+			}
+			desc = trait_creature_magnataur_character_desc
+		}
+	}
+}
+creature_furbolg = {
```
Comment (MemeWizard) [inline common/traits/wc_race_traits.txt:820]: I suggest using less tech-savy furbolg as trait icon
Current one is holding a rifle on an icon

File: common/traits/wc_race_traits.txt [hunk 4]
```diff
@@ -0,0 +1,1488 @@
...
+					NOT = { exists = this }
+				}
+				desc = trait_creature_bronze_wyrm_desc
+			}
+			desc = trait_creature_bronze_wyrm_character_desc
+		}
+	}
+}
+
+creature_giant = {
```
Comment (MemeWizard) [inline common/traits/wc_race_traits.txt:1019]: ok but
<details>
<summary>Click to expand</summary>

image

</details>

File: common/traits/wc_race_traits.txt [hunk 5]
```diff
@@ -0,0 +1,1488 @@
...
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_eredruin_desc
+			}
+			desc = trait_creature_eredruin_character_desc
+		}
+	}
+}
+creature_fel_orc = {
```
Comment (MemeWizard) [inline common/traits/wc_race_traits.txt:593]: Dark background of this icon is kinda merges with dark ck3 interface background

File: common/traits/wc_race_traits.txt (hunk 4 above)
Comment (zumbak04) [reply common/traits/wc_race_traits.txt:1019]: Changed in 63715f5fc6034c4596228752c75ab4708b7b08d9.

File: common/traits/wc_race_traits.txt (hunk 3 above)
Comment (Aruzan) [reply common/traits/wc_race_traits.txt:820]: Changed art to a more tribal looking Furbolg 460a2eaa0acff8acc5f38ca4e6188a3a7a3948f2

File: common/traits/wc_race_traits.txt (hunk 5 above)
Comment (Aruzan) [reply common/traits/wc_race_traits.txt:593]: Changed art to a lighter Fel Orc picture to cause less blending with the background 460a2eaa0acff8acc5f38ca4e6188a3a7a3948f2

File: localization/english/wc_traits_l_english.yml [hunk 6]
```diff
@@ -0,0 +1,1033 @@
...
+ trait_creature_eredruin_character_desc:0 "$trait_creature_eredruin_desc$"
+ trait_in_twisting_nether:0 "In Twisting Nether"
+ trait_in_twisting_nether_desc:0 "Demons who are killed within the mortal world are sent to the Twisting Nether, a plane for immortal beings. Demons are trapped within stasis before they are summoned into the mortal world. There is no hierarchy or name for demons in the Twisting Nether - they only go by their merit, which may or may not be needed by their dominant fel masters."
+ trait_in_twisting_nether_character_desc:0 "$trait_in_twisting_nether_desc$"
+ trait_creature_lich:0 "Lich"
+ trait_creature_lich_desc:0 "Those that ascend into high tiers of necromantic magic sacrifice themselves for unspeakable unholy power. These people are known as liches, who take on the form of ethereal, skeletal beings which hover off of the ground, commanding thralls and undead armies. Liches can be independent, or servants to the Prince of Darkness."
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply localization/english/wc_traits_l_english.yml]: Did another thing: the entire sentence didn't really make sense, so I deleted it in 908ca097e921302942998e97f94e1e357d9d5e05.

**Review by TheGroxEmpire — APPROVED** [review]

**Review by MemeWizard — APPROVED** [review]

**Pull request merged**